Traceback (most recent call last):
  File "run_joint.py", line 55, in <module>
    graph2route_pd.main(params)
  File "/home/sentanoe/joint_training/algorithm/graph2route_pd/train.py", line 232, in main
    run(params, Graph2RouteDataset, process_batch, test_model,collate_fn)
  File "/home/sentanoe/joint_training/my_utils/utils.py", line 1185, in run
    val_dataset = DATASET(mode='val', params=params)
  File "/home/sentanoe/joint_training/algorithm/graph2route_pd/model.py", line 279, in __init__
    self.unpick_x,  self.unpick_len,  self.last_x,  self.last_len,  self.eta_np,  self.order_np,  self.index_np = preprocess_food(V, V_ft, V_dt, V_dispatch_mask, cou, V_val, pad_value)
  File "/home/sentanoe/joint_training/my_utils/utils.py", line 556, in preprocess_food
    unpicked_feats = torch.cat([V_val, cou.unsqueeze(1).unsqueeze(1).repeat((1, T, N, 1))], axis=3).to(device)
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument tensors in method wrapper___cat)
srun: error: stud-000: task 0: Exited with exit code 1
